packages/vuetify/src/util/rebuildFunctionalSlots.ts

```diff
@@ -4,7 +4,7 @@
  * @param {function} h
  * @returns {array}
  */
-export default function rebuildFunctionalSlots (slots, h) {
+export default function rebuildFunctionalSlots (slots: object[], h: Function) {
```

Comment:
`h: CreateElement`, slots also should be properly typed, it's not just array of any objects